src/base/projects/ProjectRoute.svelte

```diff
@@ -12,8 +12,9 @@
   export let content: proj.ProjectContent = proj.ProjectContent.Tree;
   export let project: proj.Project;
   export let config: Config;
+  export let hash: string | null = null;
 
-  const browse: proj.BrowseTo = { content, peer, path: "/" };
+  const browse: proj.BrowseTo = { content, peer, path: "/", hash };
```

Comment:
We should parse the hash here into a line number

Reply:
Same here, eventually it wont always be a line number but could be a diff hash or another type of pointer